app/forms/field_handlers/text_area_handler.py

```diff
@@ -12,18 +13,18 @@ class TextAreaHandler(FieldHandler):
     MANDATORY_MESSAGE_KEY = "MANDATORY_TEXTAREA"
 
     @cached_property
-    def validators(self):
+    def validators(self) -> list[Union[validators.Optional, validators.Length]]:
         validate_with = super().validators
         if self.disable_validation is False:
             validate_with.append(self.get_length_validator())
         return validate_with
 
     @cached_property
-    def max_length(self):
+    def max_length(self) -> Any:
         max_length = self.answer_schema.get("max_length", 0)
         return max_length if max_length > 0 else self.MAX_LENGTH
```

Comment:
Need to avoid the use of any, this returns an int?

Reply:
Using int now, thanks!